releases/development/master/extra/install-chef-suse.sh

```diff
@@ -626,13 +627,16 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
     fi
 
     # install chef and its dependencies
-    zypper -n --gpg-auto-import-keys in rubygem-chef-server rubygem-chef rabbitmq-server \
-            couchdb rubygem-activesupport
+    zypper -n --gpg-auto-import-keys in ruby2.1-rubygem-chef-server ruby2.1-rubygem-chef \
+        rabbitmq-server couchdb ruby2.1-rubygem-activesupport
 
     # also need these (crowbar dependencies):
-    zypper -n in rubygem-app_config rubygem-cstruct rubygem-kwalify rubygem-ruby-shadow \
-            rubygem-sass rubygem-i18n sleshammer tcpdump rubygem-sprockets-helpers \
-            rubygem-sprockets-sass rubygem-sqlite3
+    zypper -n in \
+        ruby2.1-rubygem-app_config ruby2.1-rubygem-cstruct ruby2.1-rubygem-kwalify \
+        ruby2.1-rubygem-ruby-shadow ruby2.1-rubygem-sass-3_2 ruby2.1-rubygem-i18n \
+        ruby2.1-rubygem-sprockets-helpers ruby2.1-rubygem-cstruct ruby2.1-rubygem-kwalify \
+        ruby2.1-rubygem-ruby-shadow ruby2.1-rubygem-sprockets-2_11 ruby2.1-rubygem-sqlite3 \
+        sleshammer tcpdump
```

Comment:
Did you check these are the right packages to install after the rails4 port? That's the reason I didn't do anything in https://github.com/crowbar/crowbar/pull/2075 ;-)

(Btw, kwalify is there twice).